src/lib/orm/entities/StarboardEntity.ts

```diff
@@ -180,8 +178,13 @@ export class StarboardEntity extends BaseEntity {
 		const previousUpdate = this.#manager.syncMessageMap.get(this);
 		if (previousUpdate) await previousUpdate;
 
-		if ('enabled' in options) this.enabled = options.enabled!;
-		if ('stars' in options && this.enabled) await this.updateStarMessage();
+		if ('enabled' in options) {
```

Comment:
Didn't we agree we'd prefer reflect syntax? I thought we did 🙄. I mean it's not that big a deal but...

Reply:
We're switching to the type safer `isSet` from klasa-core when we do the switch.
Anyways, all the ` in ` refactors can be done later.